From b4d6713902a0c7f2c126139f1d04ff6d0b7f3241 Mon Sep 17 00:00:00 2001
From: Gao Pan <pandy.gao@nxp.com>
Date: Tue, 29 Nov 2016 14:46:14 +0800
Subject: [PATCH 1305/5242] MLK-13560 arm: imx: imx6sll: add oob support for
 sdio wifi

commit  42adcae8c7e9504219e67df1c56fd3b10952968b from
https://source.codeaurora.org/external/imx/linux-imx.git

Add oob support for imx6sll sdio wifi to improve
the performance.

Signed-off-by: Gao Pan <pandy.gao@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6sll-evk-btwifi.dts |    1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/boot/dts/imx6sll-evk-btwifi.dts b/arch/arm/boot/dts/imx6sll-evk-btwifi.dts
index 3f6b1eb..91da501 100644
--- a/arch/arm/boot/dts/imx6sll-evk-btwifi.dts
+++ b/arch/arm/boot/dts/imx6sll-evk-btwifi.dts
@@ -30,6 +30,7 @@
 	bcmdhd_wlan_0: bcmdhd_wlan@0 {
 		compatible = "android,bcmdhd_wlan";
 		wlreg_on-supply = <&wlreg_on>;
+		gpios = <&gpio3 26 0>;          /* WL_HOST_WAKE */
 	};
 };
 
-- 
1.7.9.5

